crates/torii/cli/src/options.rs

```diff
@@ -90,6 +90,10 @@ impl Default for RelayOptions {
 #[derive(Debug, clap::Args, Clone, Serialize, Deserialize, PartialEq)]
 #[command(next_help_heading = "Indexing options")]
 pub struct IndexingOptions {
+    /// If indexing should be enabled
+    #[arg(long, default_value_t = true, help = "If indexing should be enabled.")]
+    pub enabled: bool,
```

Comment:
Why would we want this to be disabled? For replicates syncing from a single indexing Torii?